Libraries/Inspector/InspectorOverlay.js

```diff
@@ -12,33 +12,27 @@
 
 const Dimensions = require('Dimensions');
 const ElementBox = require('ElementBox');
-const PropTypes = require('prop-types');
 const React = require('React');
 const StyleSheet = require('StyleSheet');
 const UIManager = require('UIManager');
 const View = require('View');
 
+import type {ViewStyleProp} from 'StyleSheet';
+
 type EventLike = {
   nativeEvent: Object,
 };
 
-class InspectorOverlay extends React.Component<{
-  inspected?: {
-    frame?: Object,
-    style?: any,
+type Props = $ReadOnly<{|
+  inspected?: ?{
+    frame?: ?Object,
```

Comment:
Let's make these nullable either. The goal of nullable props is so that a component can take optional props and pass them to another component which takes the optional prop.

Reply:
Should I make Props nullable?

```
type Props = ?$ReadOnly<{| ...
```

do not quite understand

Reply:
I'm looking for:

```
inspected?: {
  frame?: Object 
  style?: ViewStyleProp
}
```

Although inspected should also use `$ReadOnly<{||}>`. So:

```
inspected?: $ReadOnly<{|
  frame?: Object 
  style?: ViewStyleProp
|}>
```